x/superfluid/keeper/stake_test.go

```diff
@@ -1,160 +1,47 @@
 package keeper_test
 
 import (
-	"fmt"
 	"time"
 
 	abci "github.com/tendermint/tendermint/abci/types"
 
 	lockuptypes "github.com/osmosis-labs/osmosis/v7/x/lockup/types"
-	minttypes "github.com/osmosis-labs/osmosis/v7/x/mint/types"
 	"github.com/osmosis-labs/osmosis/v7/x/superfluid/keeper"
 	"github.com/osmosis-labs/osmosis/v7/x/superfluid/types"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 )
 
-type normalDelegation struct {
-	delIndex   int64
-	valIndex   int64
-	coinAmount int64
-}
-
 type superfluidDelegation struct {
 	delIndex int64
 	valIndex int64
 	lpIndex  int64
 	lpAmount int64
 }
-type superfluidRedelegation struct {
```

Comment:
currently unused, moved to the bottom with the test codes commented out for superfluid redelegation